pkg/storage/utils.go

```diff
@@ -16,6 +14,37 @@ type Query struct {
 
 type Modifier func(*Query)
```

Comment:
Now that the package name changed from `query` to `storage`, I think that this should be renamed to bring more meaning:
```suggestion
type QueryModifier func(*Query)
```
What do you think?